server/src/main/java/com/cloud/network/lb/LoadBalancingRulesManagerImpl.java

```diff
@@ -2231,11 +2235,21 @@ public Pair<List<? extends UserVm>, List<String>> listLoadBalancerInstances(List
                 continue;
             }
 
+            String userVmAsString = ReflectionToStringBuilderUtils.reflectOnlySelectedFields(userVm, "uuid", "name");
```

Comment:
As `UserVmVO` extends `VMInstanceVO` and `VMInstanceVO` has an implementation of `toString`, we could use it instead of the `ReflectionToStringBuilderUtils.reflectOnlySelectedFields`.